pom.xml

```diff
@@ -179,6 +179,12 @@
     </build>
 
     <dependencies>
+        <dependency>
+            <groupId>org.bouncycastle</groupId>
+            <artifactId>bcprov-jdk15on</artifactId>
+            <version>1.60</version>
```

Comment:
Doesn't really matter for this use case, but 1.61 is available.

Reply:
Bumped it up in latest commit